### PR TITLE
sbt build: fix `scala` command

### PR DESCRIPTION
**Pull request by SethTisue**
fooling with possible solutions to https://github.com/scala/scala-dev/issues/755, with @dwijnand's help

### CODE REVIEW

**Comment by SethTisue** [comment]
so this seems to work, but it requires you `publishLocal` first, which is a bit sad. (`disableDocs` helps with the speed.)

need to see if we can make the `publishLocal` part be triggered automatically

File: build.sbt [hunk 1]
```diff
@@ -1226,16 +1219,23 @@ watchSources ++= PartestUtil.testFilePaths((ThisBuild / baseDirectory).value, (T
...
+  val consoleCommands =
+    List(("scala",    "replRun"))
+  runCommands.map {
     case (entryPoint, projectRef, mainClassName) =>
       Command(entryPoint)(_ => ScalaOptionParser.scalaParser(entryPoint, (ThisBuild / baseDirectory).value)) { (state, parsedOptions) =>
         (projectRef + "/runMain " + mainClassName + " -usejavacp " + parsedOptions) :: state
       }
+  } ++ consoleCommands.map {
+    case (entryPoint, projectRef) =>
+      Command(entryPoint)(_ => ScalaOptionParser.scalaParser(entryPoint, (ThisBuild / baseDirectory).value)) { (state, parsedOptions) =>
```
Comment (dwijnand) [inline build.sbt:1236]: Hmmmm how do we pass the options?

**Comment by SethTisue** [comment]
Dale suggests we consider running the script in a separate process, instead of trying to duplicate the script's functionality in sbt. That way we'd be testing something closer to what users will actually use.

**Comment by SethTisue** [comment]
closing for now... keeping it on my personal todos

**Pull request closed without merging**